lib/plenario_web/templates/web/chart/show.html.eex

```diff
@@ -0,0 +1,102 @@
+<div class="container">
+  <div class="row">
+    <div class="col-lg-12">
```

Comment:
A `.row` containing only a `.col-12` just adds and then immediately removes margins and will look identical to just having the column content directly in the container. The Bootstrap grid is only necessary when you actually want grid behavior.

Also, stylistically, a lone breakpoint size like `.col-lg-12` is strange; it says "and at large size I want this to be 12 columns wide", without providing the first half of the sentence about what to do below the large breakpoint. With multiple columns in the row they'd auto-size below that. If you want to always be 12 columns it should just be `.col-12`.